src/utils/job-socket.js

```diff
@@ -8,41 +8,41 @@ export default class JobSocket extends EventEmitter {
         // setup websocket
         this._socket = new WebSocket(url);
 
-        this._socket.onopen = (event) => { this.emit('open') };
-        this._socket.onclose = (event) => { this.emit('close') };
+        this._socket.onopen = () => { this.emit("open"); };
```

Comment:
I would vote for turning that off. A callback has a signature and its signature doesn't change based on what the callback implementation actually does. I think its convenient to always have the args there so you know what you have to work with. Otherwise, every time you want to change the implementation and potentially use one of the args, you have to go look up what the signature is.

Thoughts?